### DISCUSSION TITLE
Grammatical Error?

### DISCUSSION THREAD

**Discussion by Ot92** [moment/luxon#1227] [General]
Hello, is that the right word to use? "abusing"?

https://moment.github.io/luxon/#/zones
<img width="907" alt="Screenshot 2022-07-02 at 20 17 15" src="https://user-images.githubusercontent.com/35900142/177012058-d786a1b4-2ace-467b-bb00-186ff1829052.png">

**Comment by icambron** [comment]
Yes, see definition 1a of the verb form (entry 2): https://www.merriam-webster.com/dictionary/abuse
